config/routes/user.rb

```diff
@@ -7,6 +7,7 @@
   match api_path + '/users/password_reset',        to: 'users#password_reset_send',   via: :post
   match api_path + '/users/password_reset_verify', to: 'users#password_reset_verify', via: :post
   match api_path + '/users/password_change',       to: 'users#password_change',       via: :post
+  match api_path + '/users/get_preferences',       to: 'users#get_preferences',       via: :get
```

Comment:
Don't use verbs in REST route names. Just take `/users/preferences`. The verb is defined via the HTTP method.